src/script_interface/CMakeLists.txt

```diff
@@ -12,30 +12,38 @@ set(EspressoScriptInterface_SRC
     "${CMAKE_CURRENT_SOURCE_DIR}/ScriptInterfaceBase.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/shapes/initialize.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/ParallelScriptInterface.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/ScriptInterfaceBase.cpp"
-)
+    "${CMAKE_CURRENT_SOURCE_DIR}/ScriptInterfaceBase.cpp")
 
 if(H5MD)
-    list(APPEND EspressoScriptInterface_SRC
-        "${CMAKE_CURRENT_SOURCE_DIR}/h5md/initialize.cpp"
-        "${CMAKE_CURRENT_SOURCE_DIR}/h5md/h5md.cpp"
-        )
+  list(APPEND EspressoScriptInterface_SRC
```

Comment:
I think the mordern way to do this is using [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html), which allows updating from subdirectories. I like that better because this allows to handle all source files from the CMakeLists.txt in directory they sit in. Would also be an idea for `core`.